.github/workflows/create_release.yml

```diff
@@ -0,0 +1,70 @@
+name: Create Release on PR Trigger
+on:
+  pull_request:
+    types: [closed]
+
+jobs:
+  create_release:
+    name: Windows Release
+    runs-on: windows-latest
+    if: ${{ startsWith(github.event.pull_request.title, '[Bump Version] ') }}
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 12.x
+      - run: |
+          git config --global user.email "38288322+imodeljs-admin@users.noreply.github.com"
+          git config --global user.name "imodeljs-admin[bot]"
+        name: git config
+      - run: npm ci
+      - run: npx run-s lint build:prod build:dist

```

Comment:
do you need npx for this since `npm-run-all` is a devDep?

Reply:
So you'd prefer I call `./node_modules/.bin/run-s` directly? Just because it's a dependency, doesn't mean `npm` will just find it... that's what `npx` is for. I guess I could also add a new package.json task to use `npm` instead of `npx`, but I didn't see a good reason.

Reply:
Ah, nvm, I was thinking of it as an npm script and not used directly on the cli.  I just remembered we had this same conversation last week at some point :)

Works with me then.